src/schema/items/echo.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: Echo
+class: entity
+description: |
+  Multi-echo data MUST be split into one file per echo.
```

Comment:
oh, that is a good example of an example where we have not just a description of a term ([EchoTime](https://github.com/nqueder/bids_terms_to_markdown_table/blob/master/BIDS_Terms/EchoTime.jsonld), but description there is also BIDS specific) but that it is REQUIRED (not as an entity for every file) in cases of multi-echo recording.  So now I wonder if indeed descriptions in terms should have some reflection of being BIDS-Terms ;) or we just provide descriptions like this at the level of entity etc definitions (which would also describe `format` etc) to compliment generic description of the term/concept ("The echo time (TE) for the acquisition, specified in seconds").

Reply:
yeap yeap -- the latter: term definition should be generic.  `entity` etc description would compliment with additional BIDS specific information for that "level" (term vs datatype vs suffix)!  LD people then could get a proper list of "composite" terms (e.g. entity-EchoTime and sidecar-EchoTime, although that one would unlikely have any custom description) with "composite" descriptions thus not conflicting or duplicating anything!

Reply:
oh wonderful - and validators then could validate that various instances of the term (e.g. in sidecar and filename entity) correspond since they do come from the same term, thus not hardcoding any kind of association!

Reply:
The one I'm stuck on right now is "modality". Under **Common Principles**, modality has a very nice new definition:

> Modality: the category of brain data recorded by a file. For MRI data, different pulse sequences are considered distinct modalities, such as T1w, bold or dwi. For passive recording techniques, such as EEG, MEG or iEEG, the technique is sufficiently uniform to define the modalities eeg, meg and ieeg. When applicable, the modality is indicated in the suffix. The modality may overlap with, but should not be confused with the data type.

On the other hand, the modality entity (`mod-<label>`) has its own definition, where it refers to the _corresponding_ modality for the file. The **Common Principles** definition really doesn't work for the entity, but I don't like the idea of making the principle definition less informative.

I don't think there's a good way of making one definition for both. WDYT?

Reply:
Why do you think it doesn't work for the entity, because of `When applicable, the modality is indicated in the suffix`?

Note: The only use of `_mod-` ATM is actually to absorb the suffix in MRI data whenever suffix becomes `_defacemask`, so if it works for suffix (e.g. `T1w`), it  works for the entity IMHO.

Reply:
The full definition of "modality" just doesn't seem relevant to the corresponding entity. At least it provides too much information that might confuse people. I don't want to cut it down though. I think the level of detail is necessary to understand the main concept defined in Common principles.

Reply:
I'm just going to drop Common principles from the schema for now. Since the focus on this PR is separating context from description and adding the BIDS terms, I think we can just circle around to it later.